app/assets/javascripts/chf_image_viewer.js

```diff
@@ -127,6 +129,11 @@ ChfImageViewer.prototype.selectThumb = function(thumbElement) {
   document.querySelector('*[data-hook="viewer-navbar-info-link"]').href = linkUrl;
   document.getElementsByClassName('viewer-pagination-numerator').item(0).textContent = index;
 
+  var originalLink = document.querySelector('#chf-image-viewer *[data-content-hook=dl-original-link]');
```

Comment:
`*[data-content-hook=dl-original-link]` is CSS selector for "any tag at all ("*") with attribute `data-content-hook` with value `dl-original-link`"